### ISSUE TITLE
Route Manager simulator: possibility to dynamically modify the static route table

### ISSUE THREAD

**Issue by werefin** [srsran/oran-sc-ric#6]
Hi @pgawlowicz.

My primary aim is to dynamically modify the route table obtained from the Route Manager in order to redirect traffic to a xApp for a project. The idea is to kind of replicate such attacks (as exposed [here](https://ieeexplore.ieee.org/document/10338835)) and implement a solution employing OAuth 2.0 + JWT for protection against them or similar ones. Given the static route table mentioned in the screenshot regarding all RIC entities, could this be a problem for the feasibility of my objective?

![image](https://github.com/srsran/oran-sc-ric/assets/48173952/08104216-57ea-4b27-a4de-363b9c2cfaa3)

Thanks in advance.

Best regards,
David

**Comment by pgawlowicz** [comment]
Hi, 
Not sure if the changes in the static routes are propagated after the start. 
To check this you can start some KPM monitor to receive RIC indications, change the [RIC indication route](https://github.com/srsran/oran-sc-ric/blob/main/ric/configs/routes.rtg#L16C4-L16C44) and see if the indication messages are still received in the xAPP.
Best,
Piotr

**Comment by werefin** [comment]
Hi @pgawlowicz, thanks for your response.

I'll attach below all the results I got.

![rt_debugging_1](https://github.com/srsran/oran-sc-ric/assets/48173952/778af369-4637-4886-a2f3-5cb5501e3dc2)
![rt_debugging_2](https://github.com/srsran/oran-sc-ric/assets/48173952/d3145429-ba6c-4f25-8bc4-e21bc61fe1f0)
![rt_debugging_3](https://github.com/srsran/oran-sc-ric/assets/48173952/25013d73-993c-42be-918b-200b721724b9)
![rt_debugging_4](https://github.com/srsran/oran-sc-ric/assets/48173952/beaf3ccb-cf7a-40e6-a946-0e55021be73c)

It appears that the approach you proposed is working in my case. Do you believe it's feasible to proceed with the operations I initially mentioned simply by modifying the static route table?

Thanks again.

**Comment by pgawlowicz** [comment]
Did you modify the route entries at runtime? 

If it works, then you can dynamically change routes in the RIC messaging platform (RMR). Not sure whether, based on this functionality, you can implement the proposed solution (I did not look into details).  
Note that by default, docker compose starts only one container (so one IP) for running only one xApp. If you need multiple xApps, then you probably need to start multiple containers with a different IP.

**Comment by werefin** [comment]
Yes, I modify the RIC Indication route entry at runtime. Thanks again for the help.
